.circleci/config.yml

```diff
@@ -145,7 +145,10 @@ jobs:
 
              for PKG in $PKGS ; do
                  echo_blue "===+ Building: $PKG"
-                 make "package/$PKG/compile" -j3 V=s
+                 make "package/$PKG/compile" -j3 V=s || {
+                        echo_red "===+ Building: $PKG failed, rebuilding with -j1"
+                        make "package/$PKG/compile" -j1 V=s
```

Comment:
Maybe it would better to let make fail than try again with `-j1`? The submitter should fix parallel builds or explicitly set `PKG_BUILD_PARALLEL:=0`.

Reply:
Yes, that makes more sense.
Also, no you need to specify the variable if you don't want to enable it and given the previous efforts we're not going to see that change anytime soon.

Reply:
> Maybe it would better to let make fail than try again with -j1? The submitter should fix parallel builds or explicitly set PKG_BUILD_PARALLEL:=0.

As you can see from the https://github.com/openwrt/packages/pull/9584#issuecomment-518038631 it's quite hard to find out the proper reason for the failure if you build with -j3, so this -j1 should hopefully provide more human readable error output (the same is hopefully written in the commit description).

Reply:
I see `-j1` does produce a better error message, but if compilation with `-j1` succeeds, will the CI still mark the build as failed?

Reply:
With CircleCI the separate fail logs are uploaded at the end, you just need to go look at them

Reply:
>  will the CI still mark the build as failed?

Good point, should be handled.